### ISSUE TITLE
bug ¿?

### ISSUE THREAD

**Issue by danielocampbells** [PokemonGoF/PokemonGo-Bot-Desktop#225]
![captura de pantalla 2016-08-22 a las 1 05 31 p m](https://cloud.githubusercontent.com/assets/21128626/17865742/4dedfd62-6869-11e6-9b8c-c03f1de46d23.png)
_Issue without proper information won't be answered_
### Information
- OS
- Git commit ( to get git commit run `git log -1 --format="%H"`)
- App's logs

**Comment by sunilsunny565** [comment]
Same issue. https://github.com/PokemonGoF/PokemonGo-Bot-Desktop/issues/199 Check if you can log in from device.Most probably a ban.

**Comment by mrincredible2017** [comment]
No i dont think this is a ban, happens to me at least once a day.

**Comment by danielocampbells** [comment]
i still with trouble, can login in app :/ i think that ya vali verga

**Comment by mrincredible2017** [comment]
Might be an ip ban? Are you trying to connect from a vpn or something

**Comment by roshcamlian** [comment]
almost all reports in issue about errors is normal. yes it does occur normally time to time. re-run the app.

**Comment by sunilsunny565** [comment]
In my case it was due to ban.I used a different account and it run without any issues.

**Comment by roshcamlian** [comment]
it could or could not pop out every time. so it could also be that it just did not pop out that time you used the other account. anyhow.

**Comment by sunilsunny565** [comment]
Nop I got an email from niantic saying that account is banned.And I never got this issue on the new account and got the issue every time on banned account.

**Comment by danielocampbells** [comment]
well, im still in trouble, i can't login, i uninstall the app and install again over and over, i try open in another device and nothing, i try with another account and can login, i don't got any mail from niantic saying that my account has been banned, buen i can't login, its drive me mad, also i write an email asking if i have some kind of ban, but there answer to check, but nothing happens, lol

**Comment by roshcamlian** [comment]
@sunilsunny565 oh okay.
@danielocampbells yeah I read about something like that. you could get banned even before they send you the letter. submit a ban appeal on their site.

**Comment by danielocampbells** [comment]
yeah i do that, i hope they fix my account, if no shit happens, start to 0 lol

**Comment by sunilsunny565** [comment]
@danielocampbells  I got the email a few week after this issue started .You will get an email eventually.
